src/Components/WebAssembly/Sdk/integrationtests/WasmBuildIntegrationTest.cs

```diff
@@ -335,6 +335,21 @@ public async Task Build_WithCustomOutputPath_Works()
             Assert.BuildPassed(result);
         }
 
+        [Fact]
+        public async Task Build_WithAspNetCoreFrameworkReference_Fails()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("blazorwasm-fxref");
+
+            project.AddProjectFileContent(
```

Comment:
There also is a blazorwasm-fxref.csproj in the test assets that also seems to have a package reference to JwtBearer, is that exercising something differently than here in this test?

Reply:
Nope, I missed removing this. I'll update.